node/resolvers/Routes/index.ts

```diff
@@ -310,6 +310,8 @@ export const Routes = {
       }
     }
 
+    const isCorporate = Boolean(organization)
```

Comment:
nit: won't this always be true? (as we are already checking `organization.status == active` before, so organizations is not null). If that is the case maybe we can just set this to `true` for clarity?